### PR TITLE
Added networks emotions recognition (networks dir)

### PR DESCRIPTION
**Pull request by anabwan**
added 3 files:
1- readme for emotions_detection_retail_0004
2- Makefile for downloading emotions_detection_retail_0004 network and using face_detection makefile
3- sample code (python) for emotion detection

### CODE REVIEW

**Comment by tomepetty** [comment]
Running the 'make' command gives the following error: 

emotions_recognition_retail_0003: Downloading required data...
test -f image.jpg || 
/bin/sh: 1: Syntax error: end of file unexpected
Makefile:36: recipe for target 'data' failed
make: *** [data] Error 2

I think the line "	test -f ${IMAGE} || ${GET_TEST_IMAGE}" is not working because GET_TEST_IMAGE is not initialized.

**Comment by tomepetty** [comment]
I am unable to run the command "make run". I think you are using the wrong version of the IR. The error message I am seeing is:

[ INFO ] Setting device: MYRIAD
[ INFO ] Loading Face-Detection model to the plugin
[ INFO ] Loading Emotions-Recognition model to the plugin
Traceback (most recent call last):
  File "emotions_recognition_retail_0003.py", line 140, in <module>
    sys.exit(main() or 0)
  File "emotions_recognition_retail_0003.py", line 76, in main
    emotion_exec_net = plugin.load(network=emotion_net)
  File "ie_api.pyx", line 551, in openvino.inference_engine.ie_api.IEPlugin.load
  File "ie_api.pyx", line 561, in openvino.inference_engine.ie_api.IEPlugin.load
RuntimeError: [VPU] Unsupported precision FP32

Also it looks like in the python file, it is referring to the network sample as "Face Emotion Game". Maybe this should be changed to "emotions_recognition_retail_0003".

**Comment by tomepetty** [comment]
In the Makefile and README, the input and output layer names need to be updated. I think the input layer name for this network is "data" and the output layer name is "prob_emotion".

File: networks/emotions_recognition_retail_0003/Makefile [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	@echo $(YELLOW)"  make data "$(NOCOLOR)"- Downloads the required data.";
+	@echo $(YELLOW)"  make deps "$(NOCOLOR)"- Makes dependencies for project, downloads and prepares model etc.";
+	@echo $(YELLOW)"  make install-reqs "$(NOCOLOR)"- Installs requirements needed to run this sample on your system.";
+	@echo $(YELLOW)"  make uninstall-reqs "$(NOCOLOR)"- Uninstalls requirements that were installed by the sample program.";
+	@echo $(YELLOW)"  make clean "$(NOCOLOR)"- Removes all temporary files that are created by the Makefile.\n";
+	@echo " ----- "$(YELLOW) $(NETWORK_NAME) "Model information"$(NOCOLOR)" -----"
+	@echo $(YELLOW)" Description: "$(NOCOLOR)
+	@echo "  This network does emotion recognition and can be used to align faces.";
+	@echo $(YELLOW)" Model inputs:"$(NOCOLOR)
+	@echo "  name: 'input', shape: [1x3x64x64], Expected color order is BGR.";
```
Comment (tomepetty) [inline networks/emotions_recognition_retail_0003/Makefile]: change 'input' to 'data'

File: networks/emotions_recognition_retail_0003/Makefile [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+	@echo $(YELLOW)" Model outputs:"$(NOCOLOR)
+	@echo "  name: 'detection_out', shape: [1, 5, 1, 1]  - Softmax output across five emotions ('neutral', 'happy', 'sad', 'surprise', 'anger').\n";
```
Comment (tomepetty) [inline networks/emotions_recognition_retail_0003/Makefile]: change 'detection_out' to 'prob_emotion'

File: networks/emotions_recognition_retail_0003/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+The provided Makefile does the following
+
+1. Downloads the IR files from the [Open Model Zoo](https://github.com/opencv/open_model_zoo)
+2. Takes an image and runs an inference on the emotions-recognition-retail-0003 model.
+
+The sample can also be used to crop images and write them to file. 
+
+## Model Information
+### Inputs
+ - name: "input" , shape: [1x3x64x64] - An input image in [1xCxHxW] format. Expected color order is BGR.
```
Comment (tomepetty) [inline networks/emotions_recognition_retail_0003/README.md]: Change "input" to "data"

File: networks/emotions_recognition_retail_0003/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+### Outputs 
+ - name: "prob", shape: [1, 5, 1, 1] - Softmax output across five emotions ('neutral', 'happy', 'sad', 'surprise', 'anger').
```
Comment (tomepetty) [inline networks/emotions_recognition_retail_0003/README.md]: Change "prob" to "prob_emotion"

**Comment by anabwan** [comment]
fixed..

**Pull request merged**